pkg/grpc/api_test.go

```diff
@@ -4341,7 +4341,8 @@ func TestMetadata(t *testing.T) {
 			`"subscriptions":[{"pubsub_name":"test","topic":"topic","rules":{"rules":[{"path":"path"}]},"dead_letter_topic":"dead"}],` +
 			`"http_endpoints":[{"name":"MockHTTPEndpoint"}],` +
 			`"app_connection_properties":{"port":5000,"protocol":"grpc","channel_address":"1.2.3.4","max_concurrency":10,` +
-			`"health":{"health_probe_interval":"10s","health_probe_timeout":"5s","health_threshold":3}}}`
+			`"health":{"health_probe_interval":"10s","health_probe_timeout":"5s","health_threshold":3}},` +
```

Comment:
I didn't notice this earlier, but can you please make sure that all keys use camelCase? `healthProbeInterval` etc

Reply:
This is gRPC, right? Only a part of marshaling in test. In HTTP it's still `camelCase`.

Reply:
Right!